### PR TITLE
Bugfix/celltype gene search

### PR DESCRIPTION
**Pull request by ke4**
Changes:
- Fix cell type search test case in ToBeDeprecatedJsonGeneSearchController

### CODE REVIEW

File: app/src/test/java/uk/ac/ebi/atlas/search/ToBeDeprecatedJsonGeneSearchControllerWIT.java [hunk 1]
```diff
@@ -343,14 +343,12 @@ void whenSearchTermNotExistsInDBThenCellTypeSearchReturnsEmptySet() throws Excep
 
     @Test
     void whenSearchTermExistsInDBThenReturnsSetOfCellType() throws Exception {
-        var shouldBeGeneThatPartOfExperiments = "AT2G21840";
-
-        var expectedCellType = "trichoblast 17";
+        var shouldBeGeneThatPartOfExperiments =
+                jdbcTestUtils.fetchRandomGeneFromSingleCellExperiment("E-CURD-4");
```
Comment (upendrakumbham) [inline app/src/test/java/uk/ac/ebi/atlas/search/ToBeDeprecatedJsonGeneSearchControllerWIT.java]: use can use this method to get random experiment accession - `fetchRandomExperimentAccession()`,  we have a few ways to get random experiment accession in the` JDBCutils class`, you can use one based on your need here..

**Review by upendrakumbham — CHANGES_REQUESTED** [review]
Please look at the tiny comment on randomised accession.

File: app/src/test/java/uk/ac/ebi/atlas/search/ToBeDeprecatedJsonGeneSearchControllerWIT.java (hunk 1 above)
Comment (ke4) [reply app/src/test/java/uk/ac/ebi/atlas/search/ToBeDeprecatedJsonGeneSearchControllerWIT.java]: ok, I randomised the experiment accession.

**Review by upendrakumbham — APPROVED** [review]
All good!. Looks like Jenkins build didn't pass, this test case is failed - 
```
ToBeDeprecatedJsonGeneSearchControllerWIT > jsonPayloadContainsFacetDescription() FAILED
    java.lang.AssertionError at ToBeDeprecatedJsonGeneSearchControllerWIT.java:194
```
Have a look, please. Sorry, I could have check the build before approving.

**Pull request merged**